corehq/motech/repeaters/templates/repeaters/partials/attempt_history.html

```diff
@@ -1,30 +1,69 @@
 {% load i18n %}
 
-<ul class="list-unstyled">
-{% for attempt_number, attempt in record.get_numbered_attempts %}
-    <li><strong>Attempt #{{ attempt_number }}</strong>
-    {% if attempt.state == 'SUCCESS' %}
-        <br/><i class="fa fa-check"></i>
-    {% elif attempt.state == 'EMPTY' %}
-        <br/><i class="fa fa-check"></i>
-    {% elif attempt.state == 'FAIL' or attempt.state == 'CANCELLED' %}
-        <br/><i class="fa fa-exclamation-triangle"></i>
-    {% elif attempt.state == 'PENDING' %}
-        <br/><i class="fa fa-spinner"></i>
-    {% endif %}
-    {% if attempt.state == 'PENDING' %}
-    {% elif attempt.state == 'EMPTY' %}({{ attempt.created_at }})
-    {% else %}{{ attempt.created_at }}
-    {% endif %}
-    {% if attempt.message %}
-        <a href="#" class="toggle-next-attempt">…</a>
-        <div class="well record-attempt" style="display: none; font-family: monospace;">
-            {{ attempt.message }}
-        </div>
-    {% endif %}
-    {% if attempt.state == 'CANCELLED' %}
-        <br/><i class="fa fa-remove"></i> {% trans "Cancelled" %}
-    {% endif %}
-    </li>
-{% endfor %}
-</ul>
+<tr class='attempt-row' id="{{ record_id }}">
+    <td colspan="10">
+        <table style="width:100%;">
+            {% if not has_attempts %}
+                <tr>
+                    <td>
+                        <div>
+                            {% trans 'No attempts exist for this record.' %}
+                        </div>
+                    </td>
+                </tr>
+            {% endif %}
+            {% for attempt_number, attempt in record.get_numbered_attempts %}
+                <tr>
+                    <td>
+                        <div>
+                            <strong>Attempt #{{ attempt_number }}</strong> -
+                            {% if attempt.state == 'SUCCESS' %}
+                                <i class="fa fa-check"></i>
+                            {% elif attempt.state == 'EMPTY' %}
+                                <i class="fa fa-check"></i>
+                            {% elif attempt.state == 'CANCELLED' %}
+                                <i class="fa fa-remove"></i> {% trans "Cancelled" %}
+                            {% elif attempt.state == 'FAIL' or attempt.state == 'CANCELLED' %}
+                                <i class="fa fa-exclamation-triangle"></i>
+                            {% elif attempt.state == 'PENDING' %}
+                                <br/><i class="fa fa-spinner"></i>
+                            {% endif %}
```

Comment:
```suggestion
                            {% if attempt.state == 'SUCCESS' or attempt.state == 'EMPTY' %}
                                <br/><i class="fa fa-check"></i>
                            {% elif attempt.state == 'FAIL' or attempt.state == 'CANCELLED' %}
                                <br/><i class="fa fa-exclamation-triangle"></i>
                            {% elif attempt.state == 'PENDING' %}
                                <br/><i class="fa fa-spinner"></i>
                            {% endif %}
```
See [changes for PR #32993](https://github.com/dimagi/commcare-hq/pull/32993/files).

Reply:
Updated